### ISSUE TITLE
Daily TV Show Format

### ISSUE THREAD

**Issue by maxexcloo** [itsToggle/plex_debrid#356]
I have a friend on my server who's requesting shows such as All Elite Wrestling: Rampage (https://trakt.tv/shows/all-elite-wrestling-rampage).

The problem is these shows tend not to be released as S01E01 style files, more likely to be released with a date in the release name.

Is there a way we can handle these shows?

**Comment by itsToggle** [comment]
very good idea! I'll see if I can determine "daily" shows automatically by genre. I could then add a "date" query to these shows.

**Comment by findnemo9** [comment]
+1 for this. Would be very useful. A daily-airing tv show i watch also uses dates in the filename, often like 'TV Show 24th Jan 2023' or 'TV Show 2023 01 24'

**Comment by itsToggle** [comment]
will be / is implemented in v2.66:

episodes with air date naming convention will now generally be exepted. Scraping for "The.Late.Show.With.Steven.Colbert.S08" now adds episodes named "The.Late.Show.With.Steven.Colbert.2023-01-24", "The.Late.Show.With.Steven.Colbert.24.1.2023", "The.Late.Show.With.Steven.Colbert.24.Jan.2023" and other variations.

**Comment by itsToggle** [comment]
You can also define that episodes should be scraped for with the date, rather than the S01E01 naming convention.
This only applies to episodes.
You can do this by adding a "scraping adjustment" trigger, and choosing the operator "scrape w/ airdate format". You can then define a regex definition for the show titles that this should apply to, e.g:

```
C) scraping adjustment  requirement :  scrape w/ airdate format  (late.show|daily.show|late.night)
```
